modules/react/text/lib/TypeLevelComponents.tsx

```diff
@@ -34,6 +131,35 @@ export const Heading = createComponent('h2')({
   },
 });
 
+/**
+ * ## Title
+ * [View Storybook Docs](https://workday.github.io/canvas-kit/?path=/docs/preview-text-react--title)
```

Comment:
The v8 Canvas Site link for `Title` should you choose to use it: https://canvas.workday.com/components/text/title